demos/ota/ota_demo_core_mqtt/ota_config.h

```diff
@@ -46,7 +46,7 @@
     #define LIBRARY_LOG_NAME     "OTA"
 #endif
 #ifndef LIBRARY_LOG_LEVEL
-    #define LIBRARY_LOG_LEVEL    LOG_INFO
+    #define LIBRARY_LOG_LEVEL    LOG_DEBUG
```

Comment:
Should it need to print debug information by default?

Reply:
Addressed in https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1418/commits/11afb1554eb4fcb7cbaf537332c53d27b1085317